code/src/pages/Detail.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react'
+import { Link, useParams } from 'react-router-dom'
+
+import "../css/Detail.css"
+
+const Detail = () => {
+    const [movie, setMovie] = useState ({})
+    const { id } = useParams()
+
+    useEffect (() => {
+        fetch(`https://api.themoviedb.org/3/movie/${id}?api_key=9316509df2a80192a5e837648d693f17&language=en-US`)
+            .then ((res) => res.json())
+            .then ((json) => {
+                setMovie(json)
+            })
+    }, [id])
+
+    return (
+        <article className="movieDetails">
+            <Link to="/" className="backLink">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 30 30">
+                <path d="M27 14.5C27 7.596441 21.4035594 2 14.5 2S2 7.596441 2 14.5 7.5964406 27 14.5 27 27 21.403559 27 14.5zm-19.3388348-.353553l7.4852814-7.485282c.1952622-.195262.5118446-.195262.7071068 0l2.1213203 2.121321c.1952622.195262.1952622.511844 0 .707106L12.9644661 14.5l5.0104076 5.010408c.1952622.195262.1952622.511844 0 .707106l-2.1213203 2.121321c-.1952622.195262-.5118446.195262-.7071068 0l-7.4852814-7.485282c-.19799-.19799-.197989-.509117 0-.707106z" fill="#fff" fillRule="evenodd">
+                    </path>
+            </svg>{" "}
+            Movies
+            </Link>
+            {movie && (
+                <div className="cover"
```

Comment:
Nice putting the backgroundimg in a div - we struggled a lot with getting the backgroundimg not to change size - responsivness. So it was nice to see that you solved it like this. 

And you put all the styling directly in the div... we tried to do it in the css..